src/apps/Console/Console.js

```diff
@@ -1,85 +1,48 @@
-import React, { useState, useEffect, useCallback } from 'react'
+import React, { useState, useCallback, useMemo } from 'react'
 import PropTypes from 'prop-types'
 import {
   Box,
   Header,
   Button,
   Info,
   TextInput,
+  IconEnter,
   useTheme,
   useToast,
   GU,
 } from '@aragon/ui'
-import { hash } from 'eth-ens-namehash'
+import { hash as namehash } from 'eth-ens-namehash'
+import { encodeFunctionCall } from 'web3-eth-abi'
```

Comment:
We should add `web3-eth-abi` to our package.json

Reply:
Yeah, slight mishap here; I actually installed it but due to the `this` binding bug I removed it but left that there. Added now. ✅